CTW/A_New_Day/map.json

```diff
@@ -77,8 +77,7 @@
 		}
 	],
 	"itemremove": [
-		"iron sword", "bow", "iron pickaxe", "iron axe", "leather helmet", "oak planks", "cooked beef", 
-		"arrow", "golden apple", "chainmail chestplate", "chainmail leggings", "leather boots", "beacon"
+		"iron sword", "bow", "iron pickaxe", "iron axe", "oak planks", "cooked beef", "golden apple", "chainmail chestplate", "chainmail leggings", "leather boots", "beacon"

```

Comment:
What's the reasoning for removing "arrow" and "golden apple" from the item remove? Everyone in the game is already getting 32 arrows in their inventory and getting 32 every time you kill someone probably is a bit too much. Additionally, you already get a golden apple on kill so I don't see the need to also get any golden apples the player had (every player starts with 3 golden apples, that also seems excessive).

Reply:
Got it, so if you already get a bunch of the item you don't need it. I'll do that from now on.